.env.template

```diff
@@ -93,10 +93,15 @@ OPENAI_API_KEY=your-openai-api-key
 # WEAVIATE_API_KEY=
 
 ### MILVUS
+## ZILLIZ_URI - Zilliz public endpoint (e.g. https://xxx-xxxx.xxxx.xxxx.zillizcloud.com:443)
 ## MILVUS_ADDR - Milvus remote address (e.g. localhost:19530)
```

Comment:
This is basically redundant. `Pymilvus` will determine the address as follows:
1. If `address` is set -> use that
2. If `address` is not set and `uri` is set, extract the `host:port` -> `address`
3. If `address` and `uri` are not set, use `DEFAULT_HOST:DEFAULT_PORT`

I'm fine with adding the URI config variable for convenience, and reversing the precedence of `address` and `uri`, but then name it `MILVUS_URI` and add a comment that `MILVUS_URI` takes precedence.
```suggestion
## MILVUS_ADDR - Milvus address (e.g. localhost:19530)
## MILVUS_URI - Milvus endpoint (e.g. https://xxx-xxxx.xxxx.xxxx.zillizcloud.com:443)
##   If set, MILVUS_URI overrides MILVUS_ADDR
```